migrations/000033_AddRevisionTokenKeys.up.sql

```diff
@@ -0,0 +1,27 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE RevisionKeys (
+    kid SERIAL PRIMARY KEY,
+    aad bytea NOT NULL,
```

Comment:
If we're zeroing out these, why not make them nullable and avoid the mysterious zero-byte sentinel value?